dbt_loom/__init__.py

```diff
@@ -36,13 +38,19 @@ class DbtCloudReferenceConfig(BaseModel):
     api_endpoint: Optional[str] = None
     step: Optional[int] = None
 
+class GCSReferenceConfig(BaseModel):
+    """Configuration for a GCS reference"""
+    
+    bucket_name: str
+    blob_name: str
```

Comment:
I think we should use the term `object_name` to refer to the object key instead of the term `blob_name`. I have two reasons:

1. While the GCP Storage python library uses the term `blob_name` to refer to the object key, the rest of the GCP docs use the term `object_name`.
2. Using the term `object_name` will keep us closer to the vernacular expected by AWS users.

What do you think?

Reply:
Good point ! I made the change.